app/models/user.rb

```diff
@@ -21,4 +21,12 @@ def full_name
   def proper_name
     "#{last_name}, #{first_name}"
   end
+
+  def admin_in?(roster)
+    memberships.find_by(roster: roster).admin?
```

Comment:
I guess the only thing fishy about having being an admin reliant on there being an existing roster is that any new users that want to use this service will need to have an existing admin create a roster for him and make him an admin of that. Or go through IT.